tests/backwards_compatibility_tests/base_test_case.py

```diff
@@ -0,0 +1,58 @@
+import json
+import logging
+import os
+import sys
+from pathlib import Path
+
+from marqo_test import MarqoTestCase
+
+sys.path.append(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
```

Comment:
what does this do?

Reply:
this particular instance wasn't needed - so I removed it